### PR TITLE
OSDOCS-3428: Adds clarity to multicluster release note

### PR DESCRIPTION
**Pull request by opayne1**
[OSDOCS:3428](https://issues.redhat.com/browse/OSDOCS-3428)

- Applies to `enterprise-4.10` only.
- [Preview link](http://file.rdu.redhat.com/opayne/console-RN-clarification/release_notes/ocp-4-10-release-notes.html#ocp-4-10-multicluster-console-technology-preview)

### CODE REVIEW

**Comment by opayne1** [comment]
Closing for the build to run

**Comment by opayne1** [comment]
This change should follow change management, so I am sending out an email to stakeholders soon, and also tagging relevant people here:

- TAM: @sreber84 
- PM: @spadgett/ @alimobrem 
- PX: @sferich888 
- QE: @yapei 
- Docs: @ahardin-rh

**Comment by ahardin-rh** [comment]
LGTM!

**Comment by sreber84** [comment]
I understand either Advanced Cluster Manager or MCE (Multicluster Engine) is required. Not sure if MCE is already available but still wondering if that should be somehow mentioned.

Otherwise LGTM

**Comment by yapei** [comment]
LGTM

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -246,7 +246,7 @@ For more information about the dynamic plug-in, see xref:../web_console/dynamic-
 
 [id="ocp-4-10-multicluster-console-technology-preview"]
 ==== Multicluster console (Technology Preview)
-Starting with {product-title} 4.10, the multicluster console is now available as a Technology Preview feature. By enabling this feature, you can connect to remote clusters' API servers from a single {product-title} console.
+Starting with {product-title} 4.10, the multicluster console is now available as a Technology Preview feature. By enabling this feature, you can connect to remote clusters' API servers from a single {product-title} console. You must have Red Hat Advanced Cluster Management (ACM) installed to enable this feature.
```
Comment (spadgett) [inline release_notes/ocp-4-10-release-notes.adoc:249]: I think we want to mention MCE as an alternative, although the operator is not yet available. cc @showeimer

Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc:249]: Should we include this for now and then mention MCE when it becomes available with ACM 2.5? @spadgett @showeimer

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -251,7 +251,7 @@ For more information about the dynamic plug-in, see xref:../web_console/dynamic-
 
 [id="ocp-4-10-multicluster-console-technology-preview"]
 ==== Multicluster console (Technology Preview)
-Starting with {product-title} 4.10, the multicluster console is now available as a Technology Preview feature. By enabling this feature, you can connect to remote clusters' API servers from a single {product-title} console.
+Starting with {product-title} 4.10, the multicluster console is now available as a Technology Preview feature. By enabling this feature, you can connect to remote clusters' API servers from a single {product-title} console. You must have Red Hat Advanced Cluster Management (ACM) or the multicluster engine (MCE) operator installed to enable this feature.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:254]: Operator

Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc]: Updated! Thank you for the reminder!

**Review by TheRealJon — APPROVED** [review]
/lgtm

**Pull request merged**